examples/astro/README.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to [Astro](https://astro.build)
```

Comment:
We might remove the default readme, but instead, explain about how to integrate UnoCSS intro Astro and with the caveats you mentioned.